server/server_test.go

```diff
@@ -559,6 +559,34 @@ var _ = Describe("Server package test", func() {
 					testURL("PUT", url, memberTokenID, testResource, http.StatusUnauthorized)
 				})
 			})
+
+			Context("Creating resource in different tenant", func() {
```

Comment:
I've checked that the validation is also applied in case of update but as the different piece of code is responsible for updating and creation resources I think it's good idea to cover the case of update also in the unittests.

Reply:
in common case tenant/domain id are not properties for which update is possible - besides it's the same logic so i don't see any added value in this scenario